apps/condo/domains/banking/schema/BankAccount.test.js

```diff
@@ -252,6 +254,47 @@ describe('BankAccount', () => {
                 expect(readObjects).toHaveLength(0)
             })
 
+            test('resident user can read for its property', async () => {
```

Comment:
`resident user can read for its property`

I guess there's a typo, something is missing